pages/api/Schema/TaskAnalytics/taskAnalytics.graphql

```diff
@@ -13,8 +13,6 @@ type TaskAnalytics {
   updatedInDbAt: ISO8601DateTime!
 }
 
-scalar ISO8601DateTime
```

Comment:
Oh yeah, looks like it was because it was declared in more than one spot. I think extracting to a separate file like you did was a good idea. Question is, do we want a specific file/folder for custom types or just one common folder that holds everything? Not sure how many other custom scalars we actually have...

Reply:
These are the only custom scalars that I know of... I think the default types are good enough for most of our use cases.